homeassistant/components/lupusec/__init__.py

```diff
@@ -56,7 +55,7 @@ def setup(hass: HomeAssistant, config: ConfigType) -> bool:
 
     try:
         hass.data[DOMAIN] = LupusecSystem(username, password, ip_address, name)
-    except LupusecException as ex:
+    except Exception as ex:
```

Comment:
Why should we change this? We want to avoid catching broad exception. It's best if the library keeps catching transport specific exceptions and raising library specific exceptions.

Reply:
I reimplemented the exceptions. Is there a way to re-trigger the pipeline manually?

Reply:
The build passed now.